WootricSDK/WootricSDK/WTRSettings.m

```diff
@@ -94,6 +109,34 @@ - (void)parseDataFromSurveyServer:(NSDictionary *)surveyServerSettings {
   }
 }
 
+- (NSDictionary *)scoreRules {
+    return @{
+             @"NPS" : [NSArray arrayWithObjects: @{@"min" : [NSNumber numberWithInt:0], @"max" : [NSNumber numberWithInt:10], @"negative_type_max" : [NSNumber numberWithInt:6], @"neutral_type_max" : [NSNumber numberWithInt:8]}, nil],
```

Comment:
The array is for the survey_type_scale argument which comes from the survey server. For example, CSAT has two entries in its array of scales, so we could be using either the first or second.